spec/models/google_play_store_integration_spec.rb

```diff
@@ -196,4 +205,56 @@
       expect(google_integration.build_in_progress?("track", 1)).to be false
     end
   end
+
+  describe "#api_lock" do
```

Comment:
These tests are a little unorthodox, but testing distributed locks and its behavior is quite tricky in a unit-way.